app/views/posts/show.html.erb

```diff
@@ -20,5 +20,9 @@
   <%= @post.formatted_document %>
 </p>
 
+<% if !@post.event.nil? && !@post.event.event_type.nil? %>
```

Comment:
We can use the safe navigation operator here too.
Ruby evaluates strings to true so we van use:
```suggestion
<% if @post.event&.event_type %>
```